feature-libs/product-configurator/rulebased/components/default-form/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export * from './configurator-default-form.component';
+export * from './configurator-default-form.event';
```

Comment:
Why do we have to move the event? Having it at the same location saves us many changes in this PR

Reply:
For the sake of completeness, I will say. Both `form` and `conflict-solver-dialog` inject `default-form`. According to my point of view, it could be questionable if the event stays in `form` and maybe one could ask oneself why not in `conflict-solver-dialog`. We came up to the conclusion that the common part could be extracted the from the form and the event should belong to this common part as well.